src/pybind/mgr/influx/module.py

```diff
@@ -55,6 +58,7 @@ def get_df_stats(self):
                         "pool_id" : pool['id'],
                         "type_instance" : df_type,
                         "mgr_id" : self.get_mgr_id(),
```

Comment:
I just noticed this rogue `mgr_id` label here, let's remove this while we're updating labels